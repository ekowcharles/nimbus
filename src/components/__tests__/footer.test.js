import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Footer from '../footer'

describe('Footer', () => {
  it('renders footer with terms of service and privacy information', () => {
    const wrapper = shallow(<Footer />)

    expect(toJson(wrapper)).toMatchSnapshot()
  })
})