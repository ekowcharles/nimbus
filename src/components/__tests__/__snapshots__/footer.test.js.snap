// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders footer with terms of service and privacy information 1`] = `
<div
  className="area-footer smaller"
>
  © Ovsprint LLC. · All rights reserved
  <br />
  <br />
  <span
    className="clickable smallest pointer"
    onClick={[Function]}
  >
    Terms of Service
  </span>
    |  
  <span
    className="clickable smallest pointer"
    onClick={[Function]}
  >
    Privacy Policy
  </span>
</div>
`;
